components/Dynamic/DynamicComponents.tsx

```
@@ -1,164 +1,321 @@
+    high: 'animate-[pulse_1s_ease-in-out_infinite]'
+  };
+
+  return (
+    <div
+      className={cn(
+        'bg-muted rounded-md',
+        priorityStyles[priority],
+        'motion-reduce:animate-none',
+        className || 'h-32'
+      )}
+      role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)